apps/mark/frontend/src/pages/replace_election_screen.tsx

```diff
@@ -30,31 +30,48 @@ export function ReplaceElectionScreen({
   appPrecinct,
   ballotsPrintedCount,
   electionDefinition,
-  electionDefinitionFromCard,
   machineConfig,
   screenReader,
   unconfigure,
-}: Props): JSX.Element {
+}: ReplaceElectionScreenProps): JSX.Element {
   const { election, electionHash } = electionDefinition;
+  const electionDefinitionFromCardQuery =
+    getElectionDefinitionFromCard.useQuery(electionHash);
+  const electionDefinitionFromCard = electionDefinitionFromCardQuery.data?.ok();
 
   useEffect(() => {
     const muted = screenReader.isMuted();
     screenReader.mute();
     return () => screenReader.toggleMuted(muted);
   }, [screenReader]);
 
-  if (!electionDefinitionFromCard) {
+  if (!electionDefinitionFromCardQuery.data) {
```

Comment:
Generally I like to use `!query.isSuccess` to be more precise about what I'm checking, since the `data` might well be falsy for some queries.